loadtest/user/userentity/actions.go

```diff
@@ -647,6 +655,15 @@ func (ue *UserEntity) SaveReaction(reaction *model.Reaction) error {
 	return nil
 }
 
+func (ue *UserEntity) DeleteReaction(reaction *model.Reaction) error {
+	_, resp := ue.client.DeleteReaction(reaction)
```

Comment:
`DeleteReaction` returns a boolean that maybe we should also check an return error if its false

Reply:
If there's an error with the response that will always be false. Otherwise it should be true. I don't think we need to explicitly check for that.